wp-content/plugins/goonj-blocks/src/render.php

```diff
@@ -3,6 +3,8 @@
  * @see https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-metadata.md#render
  */
 
+define('DATE_FORMAT', 'd-m-Y');
+define('TIME_FORMAT', 'h:i A');
```

Comment:
Remove these – these names suggests that we are setting global date and time format.

Reply:
sure